src/tsolvers/egraph/Enode.h

```diff
@@ -33,6 +33,7 @@ WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
 #include "PtStructs.h"
 #include "SymRef.h"
 #include "TypeUtils.h"
+#include "Global.h"
```

Comment:
Hopefully we do not need this include?

Reply:
This is needed because of the `SET_BIT` thing, I guess. Could we avoid this somehow?

Reply:
Fixed.